tox.ini

```diff
@@ -51,9 +51,11 @@ commands =
   coverage combine
   coveralls
 
-[testenv:pep8]
-deps = pep8
-commands = pep8 --exclude=luigi/six.py --ignore E501,E402,E731 luigi test examples bin
+[testenv:flake8]
+deps = flake8
+commands = flake8 --exclude=doc,test,luigi/six.py --ignore=E501
+  flake8 --ignore=E265,E501 doc
+  flake8 --ignore=E501,F401,F841 test
```

Comment:
Instead of ignoring E501 can you just set the line length limit to the longest line used?